### ISSUE TITLE
video.js.map 404

### ISSUE THREAD

**Issue by BrysonR** [SamyPesse/tv.js#37]
I'm getting a 404, file cannot be located, error in regards to video.js.map. Is this the reason I'm having difficulty downloading anything?

**Comment by SamyPesse** [comment]
No, this 404 error is not important.
Do you have a correct version of ffmpeg ?
Check on your terminal if the server logs downloads using torrents.

(Sorry for my english)

**Comment by BrysonR** [comment]
my console is displaying update log streaming stats that are json objects but I'm not actually seeing anything going on in terminal. is that what you are referring to?

**Comment by SamyPesse** [comment]
When you click on play on the movie you should have an output like this :
http://i.imgur.com/Hcd42ns.png

**Comment by BrysonR** [comment]
hmmm. I'm definitely not getting that. any suggestions as to an easy movie to test it with?

**Comment by BrysonR** [comment]
I didn't mean to close this oops

**Comment by SamyPesse** [comment]
No problem :)
Test with 'harry potter 6", it's working for me.
And test in a command line to run the command "ffmpeg"

**Comment by BrysonR** [comment]
yep. Doesn't seem to be working with Harry Potter 6 either. I ran ffmpeg in my terminal and it's telling me that it doesn't exist...gah. and then to make things more fun, i type sudo port install ffmpeg and it says it's installed fine.

**Comment by SamyPesse** [comment]
There is a section avout how to install ffmpeg on Mac OS X in the readme
suing brew.

**Comment by BrysonR** [comment]
I attempted to install it that way and I can't seem to get it to finish without errors. Always stops on SDL. I think there's a good chance that it is because I have the new Mavericks 10.9 OS X installed which i don't have the command line tools installed for yet so i'm getting those now to see if that's the issue.
